src/surge-xt/gui/widgets/OscillatorWaveformDisplay.cpp

```diff
@@ -573,15 +573,15 @@ void OscillatorWaveformDisplay::populateMenu(juce::PopupMenu &contextMenu, int s
     }
 
     auto rnaction = [this]() {
-        char c[TXT_SIZE];
-        strncpy(c, this->oscdata->wavetable_display_name, TXT_SIZE);
+        std::string c;
+        c = this->oscdata->wavetable_display_name;
```

Comment:
this could be `auto c = this->oscdata` but only if we find things that require changes otherwise its ok